graph/graph.go

```diff
@@ -104,6 +105,20 @@ func (g *Graph) Importers(ctx context.Context, pkg string, f func(string)) error
 	})
 }
 
+// ImportersScan calls f with the import path of each package that directly depends
+// on anything with pkg prefix. The order of results is unspecified.
+func (g *Graph) ImportersScan(ctx context.Context, pkg string, f func(string)) error {
```

Comment:
This is a good suggestion! I addressed it in a somewhat different way in 48914d14, does that meet your needs?